yo/api_server.py

```diff
@@ -80,15 +80,18 @@ class YoAPIServer(YoBaseService):
            ids(list): List of notifications to mark read
        
        Returns:
-           list: list of notifications updated
+           dict: single key 'success' with boolean value
        """
-        retval = []
-        if test and self.testing_allowed:
-            for notify_id in ids:
-                self.test_notifications.mark_notification_read(notify_id)
-                retval.append(
-                    self.test_notifications.get_notification(notify_id))
-        return retval
+        success = False
+        if test:
```

Comment:
this "test" nonsense should not be in the production code

Reply:
I've been over this multiple times with @john-g-g and pointed out that if removed this will only slow down development and having it in place will not harm anything.

The testing_allowed flag can be disabled by default for production code, but it is VERY useful for development and maintaining it in a separate branch etc overcomplicates matters.